demos/common/network_manager/aws_iot_network_manager.c

```diff
@@ -57,12 +61,12 @@
 /**
  * @brief Initializer for WiFi Network Info structure.
  */
-#define _NETWORK_INFO_WIFI            { .ulNetworkType = AWSIOT_NETWORK_TYPE_WIFI, .xNetworkState = eNetworkStateUnknown }
+#define _NETWORK_INFO_WIFI            { .ulNetworkType = AWSIOT_NETWORK_TYPE_WIFI, .xNetworkList = IOT_LINK_INITIALIZER, .xNetworkState = eNetworkStateUnknown }
```

Comment:
per coding conventions, no Hungarian notation. So please remove the 'x', 'ul' and so on.... look at task pol or any mqtt file to get an idea...

Reply:
Updated in PR https://github.com/aws/amazon-freertos/pull/462